axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/batch/BatchBillOfExchange.java

```diff
@@ -124,27 +125,44 @@ protected void process() {
   protected void createLCRAccountingMovesForInvoices(
       Query<Invoice> query, List<Long> anomalyList, AccountingBatch accountingBatch) {
     List<Invoice> invoicesList = null;
-    while (!(invoicesList = query.fetch(FETCH_LIMIT)).isEmpty()) {
+    while (!(invoicesList = query.bind("anomalyList", anomalyList).fetch(FETCH_LIMIT)).isEmpty()) {
       for (Invoice invoice : invoicesList) {
         try {
           createMoveAndUpdateInvoice(accountingBatch, invoice);
           incrementDone();
         } catch (Exception e) {
           anomalyList.add(invoice.getId());
-          query.bind("anomalyList", anomalyList);
           incrementAnomaly();
           TraceBackService.trace(
               e, "billOfExchangeBatch: create lcr accounting move", batch.getId());
-          break;
+          continue;
         }
       }
       JPA.clear();
     }
   }
 
-  @Transactional(rollbackOn = {Exception.class})
```

Comment:
why do we need to remove this transactional ? We need at least a transaction for the unitary process, else on error we could generate moves without updating the invoice.